apps/site/components/Common/Select/index.module.css

```diff
@@ -60,7 +60,7 @@
 
 .dropdown {
   @apply max-h-48
-    max-w-xs
+    max-w-fit
```

Comment:
```suggestion
    max-w-xs
```
I think it seems more appropriate to leave the width of these areas as is

[Current behavior](https://64c7d71358830e9105808652-gecygyftxp.chromatic.com/?path=/story/common-select--default&globals=viewport:large);
<img width="642" alt="image" src="https://github.com/user-attachments/assets/a7341f14-28fc-4743-a93f-235f8e74d818">

[New behavior](https://64c7d71358830e9105808652-erkbtwjkcu.chromatic.com/?path=/story/common-select--default&globals=viewport:large);
<img width="642" alt="image" src="https://github.com/user-attachments/assets/9bbda8a5-8793-4ddd-a6b7-c0442dd7949c">

It seems like a better experience to be able to navigate between options without moving the mouse horizontally after clicking on a select

Reply:
that look better, thanks for pointing that out !! 
have made the changes cc @canerakdas